Realm/Realm.SourceGenerator/Parser.cs

```diff
@@ -145,7 +147,7 @@ private static IEnumerable<string> GetUsings(ClassDeclarationSyntax classDeclara
             return usings;
         }
 
-        private static IEnumerable<PropertyInfo> GetProperties(ClassInfo classInfo, IEnumerable<PropertyDeclarationSyntax> propertyDeclarationSyntaxes, SemanticModel model)
```

Comment:
Why did you remove the static keyword? It does not look clear to me as the method does not look changed.

Reply:
This method calls `GetPropertyTypeInfo ` that is not static anymore 😁